### ISSUE TITLE
Dialog doesn't look good when the text is too long and the close button on the top right does not work

### ISSUE THREAD

**Issue by kimlimjustin** [PolyMeilex/rfd#34]
![image](https://user-images.githubusercontent.com/61906684/147037896-11bcbafc-50d6-411c-8562-abe624dc99b6.png)

I encountered this error when testing [Tauri](https://github.com/tauri-apps/tauri)'s auto-updater which uses rfd's dialog, the ideal would be make it scrollable when the text is too long, and if possible, make the link is clickable and functioning.

Thanks in advance!

**Comment by PolyMeilex** [comment]
Hey!
The thing is, we don't do our own dialogs, we just tell windows to open one, so it's up to windows to decide how to display it, and how to break its functionality randomly :smile: 

But nonetheless the "no close button" is alarming, do you think you could somehow test it with shorter messages to see if it works at all?

**Comment by kimlimjustin** [comment]
> Hey! The thing is, we don't do our own dialogs, we just tell windows to open one, so it's up to windows to decide how to display it, and how to break its functionality randomly 😄

Ah I see, but is there possible to make it scrollable? or is it windows thing too?

> 
> But nonetheless the "no close button" is alarming, do you think you could somehow test it with shorter messages to see if it works at all?

Yes, not working at all.

**Comment by PolyMeilex** [comment]
Could you try with rfd examples
just `cargo run --example msg`

**Comment by kimlimjustin** [comment]
> Could you try with rfd examples just `cargo run --example msg`

Yes, it's working fine, but not with Tauri, so issit Tauri's problem (I guess too)

**Comment by PolyMeilex** [comment]
Hard to tell where the problem is, but maybe it has something to do with the fact that Tauri puts the dialog in a new thread? Just a random guess.

**Comment by kimlimjustin** [comment]
Tbh I got no idea bout it 😂
